clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -136,7 +137,12 @@ export class LanguageClientService {
         fileEvents: [
           vscode.workspace.createFileSystemWatcher("**/pgm_conf.json"),
           vscode.workspace.createFileSystemWatcher("**/proc_grps.json"),
-          vscode.workspace.createFileSystemWatcher("**/.copybooks/**/*"),
+          vscode.workspace.createFileSystemWatcher(
+            new vscode.RelativePattern(
+              vscode.Uri.file(Utils.getExtensionFolder()),
```

Comment:
Again, could be just a parameter of `createClientOptions`.

Reply:
resolved at : 224e9b8e00a6753439dd54d8da5d6df093b6cf89